src/controllers/user.controllers.js

```diff
@@ -33,8 +34,42 @@ export const userLogin = async (data) => {
   const validPass = await bcrypt.compare(data.password, user.password);
   if (!validPass) return { status: 'invalid', message: 'Email or password is wrong' };
 
-  const token = jsonwebtoken.sign({_id:  user._id}, process.env.TOKEN_SECRET)
-  response.header('auth-token', token);
+  const token = jsonwebtoken.sign({_id: user._id}, process.env.TOKEN_SECRET)
+  const header = ('auth-token', token);
 
   return `Witam ${user.name}!`;
 };
+
+export const userEdit = async (req) => {
+  const { error } = editValidation(req.body);
+  if (error) return { status: 'invalid', message: error.details[0].message };
+
+  updateUser(req);
```

Comment:
`updateUser` has a return statement that doesn't affect the logic of `userEdit` because it's not assigned to any variable and used thereafter so it's possible to send a request with a user that doesn't exist and get the message 'User updated'. When trying to do the same second time, there's a server error. The solution could be to move the statement `return { message: 'User updated' };` inside `updateUser` and then `return updateUser;` inside `userEdit`.